docs/en_US/reference/experiment_config.rst

```diff
@@ -464,10 +476,10 @@ type: ``Optional[str]``
 This is useful if preparing steps vary for different machines.
 
 
-OpenPaiConfig
+OpenpaiConfig
```

Comment:
PAI's official doc use the name of `OpenPAI`, [refer]( https://github.com/microsoft/pai#when-to-consider-openpai)

Reply:
This is the Python class name, so `OpenPAIConfig` violates PEP 8.